client/app/user/profile.html

```diff
@@ -3,6 +3,9 @@
   <form>
     <input type="text" ng-model="hero.author" /></h2>
     <textarea ng-model="hero.bio" />
+      <select class="u-full-width" id="exampleRecipientInput">
+        <option ng-repeat="theme in themes"> {{ theme }} </option>
```

Comment:
This is the json object I am planning to return:

{ "default": "filepathtostyles.csshere" , "cheetos": "filepathtocheetos.csshere"}

so theme in themes would be:
default, cheetos

let me know if u guys think otherwise.